bot-get.js

```diff
@@ -31,4 +31,124 @@ There will be multiple "regions" and "locations", but the Progression region wil
 We can guarantee a 100% chance.
 
 
-*/
\ No newline at end of file
+*/
+
+var goal = false;
+
+var server = require('./server.js');
+var webhook = require('./webhook-put.js');
+var chatipelago = require('./archipelagoHelper.js');
+var config = require('./config.js');
+
+server.setOnEvent(onEvent);
+chatipelago.setOnItemRecieved(onItem);
+
+function onEvent(message) {
+    chatipelago.maybeTriggerItemLocationMap();
+    switch (message) {
+        /*case '/table':
+            attemptClaimTable();
+            break;
+        case '/button':
+            attemptPressButton();
+            break;*/
```

Comment:
We can probably erase these code snipped and not just comment them out at this point